src/zui/ZUICleanHtml.tsx

```diff
@@ -11,7 +12,7 @@ const ZUICleanHtml = ({
   BoxProps,
   dirtyHtml,
 }: ZUICleanHtmlProps): JSX.Element => {
-  const cleanHtml = DOMPurify.sanitize(dirtyHtml);
+  const cleanHtml = useMemo(() => DOMPurify.sanitize(dirtyHtml), [dirtyHtml]);
   return <Box dangerouslySetInnerHTML={{ __html: cleanHtml }} {...BoxProps} />;
```

Comment:
You are correct that `Box` will re-render whenever `ZUICleanHtml` re-renders. Is that a problem in your opinion? It won't change the DOM though.

Personally I'm not entirely sure how `dangerouslySetInnerHTML` is implemented internally in React, but I don't expect the DOM to change if the HTML value did not change, so re-rendering the box should be entirely negligible.

The goal of using `useMemo()` here is not to avoid re-rendering, but to avoid re-sanitizing the HTML (which is a heavier operation).